### PR TITLE
Add impact qualification

### PR DESCRIPTION
**Pull request by monsieurswag**

### CODE REVIEW

File: backend/core/models.py [hunk 1]
```diff
@@ -1680,6 +1680,23 @@ class RiskScenario(NameDescriptionMixin):
...
 
+    QUALIFICATIONS = [
+        ("Financial", _("Financial")),
+        ("Legal", _("Legal")),
+        ("Reputation", _("Reputation")),
+        ("Operational", _("Operational")),
+        ("Confidentiality", _("Confidentiality")),
+        ("Integrity", _("Integrity")),
+        ("Availability", _("Availability"))
+    ]
```
Comment (nas-tabchiche) [inline backend/core/models.py]: We should allow impact qualifications to be specified in risk matrices, as it is done for strength of knowledge. Most likely some users will want to have their own

Comment (eric-intuitem) [reply backend/core/models.py]: The default values can be what is proposed here.

**Review by eric-intuitem — CHANGES_REQUESTED** [review]
1/ The qualification can be multivalued (eg financial + confidentiality).
2/ I The addition of the status is useless, and in fact the added status is dead code.

**Review by eric-intuitem — CHANGES_REQUESTED** [review]
Almost good! Just need to translate the qualification values.

**Review by eric-intuitem — APPROVED** [review]
LGTM

**Pull request merged**